app/src/main/java/org/groundplatform/android/persistence/remote/firebase/schema/SurveysCollectionReference.kt

```diff
@@ -36,10 +37,17 @@ class SurveysCollectionReference internal constructor(ref: CollectionReference)
 
   fun getReadable(user: User): Flow<List<Survey>> =
     reference()
+      .whereEqualTo(GENERAL_ACCESS_FIELD, SurveyProto.GeneralAccess.RESTRICTED_VALUE)
```

Comment:
While testing locally, I found that this field was introduced later and it caused my previous surveys to not get listed. 

@gino-m Is this acceptable behavior or do we need to handle backward compatibility?

Reply:
No, we can't break existing surveys.

@anandwana001 Can you please use `.whereIn` instead and allow UNSPECIFIED.. as well as RESTRICTED_VALUE?

Reply:
Would it be a better approach if we set the default value in proto itself rather than handling it separately in both android app and web client? @gino-m

Reply:
But I don't know if that'll work for already created surveys. Let's stick to our current approach of explicitly handling unspecified value. Maybe a food for thought when introducing new fields in the future

Reply:
I ran the latest branch at HEAD and this problem still exists even with `whereIn`. The older surveys don't even have the proto field available.

@gino-m @anandwana001 fyi

Reply:
Can you help me with which survey you are not seeing?

I tested on one of my old survey, It is coming in my list.

Reply:
In order to test this, choose one of your older surveys and remove the corresponding `GENERAL_ACCESS_FIELD` from the remote firebase db. Then restart your app and try again